guides/common/modules/proc_configuring-satellite-repositories.adoc

```diff
@@ -0,0 +1,70 @@
+.Procedure
+Select the operating system and version you are installing on:
+
+* xref:#repositories-el-9[{EL} 9]
+* xref:#repositories-el-8[{EL} 8]
+
+[id="repositories-el-9"]
+== {EL} 9
+
+. Disable all repositories:
++
+[options="nowrap"]
+----
+# subscription-manager repos --disable "*"
+----
++
+. Enable the following repositories:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos \
+--enable={RepoRHEL9BaseOS} \
+--enable={RepoRHEL9AppStream} \
+--enable={RepoRHEL9ServerSatelliteServerProjectVersion} \
+--enable={RepoRHEL9ServerSatelliteMaintenanceProjectVersion}
+----
+
+include::snip_verification-enabled-repolist.adoc[]
+
+[id="repositories-el-8"]
+== {EL} 8
+
+. Disable all repositories:
++
+[options="nowrap"]
+----
+# subscription-manager repos --disable "*"
+----
++
+. Enable the following repositories:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos \
+--enable={RepoRHEL8BaseOS} \
+--enable={RepoRHEL8AppStream} \
+--enable={RepoRHEL8ServerSatelliteServerProjectVersion} \
+--enable={RepoRHEL8ServerSatelliteMaintenanceProjectVersion}
+----
+
+. Enable the DNF modules:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# dnf module enable {dnf-module}
+----
++
+[NOTE]
+====
+If there is any warning about conflicts with Ruby or PostgreSQL while enabling `{dnf-module}` module, see
+ifeval::["{context}" == "{project-context}"]
+xref:troubleshooting-dnf-modules_{context}[].
+endif::[]
+ifeval::["{context}" != "{project-context}"]
+{InstallingServerDocURL}troubleshooting-dnf-modules_{project-context}[Troubleshooting DNF modules] in _{InstallingServerDocTitle}_.
+endif::[]
+For more information about modules and lifecycle streams on {RHEL} 8, see https://access.redhat.com/support/policy/updates/rhel-app-streams-life-cycle[{RHEL} Application Streams Lifecycle].
+====
```

Comment:
This note is duplicated for `upstream-rpm`. I think that's acceptable and Foreman 3.13 will drop EL8 support anyway. For now we don't have modules on EL 9.